### DISCUSSION TITLE
Green cursor-keyboard inactive in code editor

### DISCUSSION THREAD

**Discussion by codeviewnext** [codewars/codewars.com#2461] [Q&A] [answered]
Hello 

  I use codewars as training platform to excercise coding in python.
Lately i have problem in codewars code editor. Cursor appears as green blinking rectangular and I cannot use my keyboard.
During typing  this cursor sometimes is moving and looks like being mark parts of text.
Sometimes after typing sequence of random keys it is going back to normal.

Any idea how to fix it?

**Comment by hobovsky** [comment, marked as answer]
You've probably switched the editor into vim mode: https://docs.codewars.com/references/kata-trainer#editors

Toggle the editor mode off and things should work like before.

**Reply by codeviewnext** [reply]
This is it. Thanks!

**Reply by skankster** [reply]
Thanx!
